test/com/facebook/buck/cxx/toolchain/WindowsPreprocessorTest.java

```diff
@@ -0,0 +1,60 @@
+package com.facebook.buck.cxx.toolchain;
+
+import static org.hamcrest.Matchers.contains;
+import static org.junit.Assert.assertThat;
+
+import com.facebook.buck.rules.Tool;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.easymock.EasyMock;
```

Comment:
Please make sure this builds with buck
https://travis-ci.org/facebook/buck/jobs/370932370#L4067

you need to add `"//third-party/java/easymock:easymock"`to the deps of the BUCK file for this test.

Reply:
Oh, okay. I didn't know there are separate `BUCK` files for each test package. Seems like a bit of an overkill to me. But sure, will do